pyuvdata/uvdata/uvdata.py

```diff
@@ -12280,10 +12352,13 @@ def read(
         phase_to_pointing_center=False,
         nsample_array_dtype=np.float32,
         # MIR
-        isource=None,
-        irec=None,
-        isb=None,
```

Comment:
I don't think so -- these were basically options for some advanced users to be able to conduct testing, but I don't know of anyone actually using them on the regular (since they're a bit... esoteric). I'd be in favor of clearing them out completely just to avoid confusion once we start broadcasting support for MIR via pyuvdata more broadly.